src/Identifier/Ldap/AdapterInterface.php

```diff
@@ -35,6 +35,15 @@ public function bind($bind, $password);
      */
     public function connect($host, $port, $options);
 
+    /**
+     * Search the LDAP directory
+     *
+     * @param string $baseDN Base DN for the directory
+     * @param string $filter search filter
+     * @return array
+     */
+    public function search($baseDN, $filter);
```

Comment:
What does DN mean?
Usually, variables are called `$baseDn` or the long version.

Reply:
DN is an initialism for Distinguished Name.